### ISSUE TITLE
Okular

### ISSUE THREAD

**Issue by cbcoutinho** [rudrab/Shadow#70]
Okular used to work fine, but now it's having issues on Gnome (version 3.22 I believe)

The icon works fine in the application menu, but in the gnome status bar and the active window bar (the one you use cmd+tab), the Okular icon is just a grey blob. Actually, everywhere _except_ in the the application menu, it's a grey blob

Not sure what happened, it could be an incompatible break within Okular recently. Any thoughts?

**Comment by rudrab** [comment]
Hi, If you are seeing this behaviour, you are probably in wayland session. I have faced such things with libreoffice as well. Can you let me know if this is only for Shadow, or you see same bahaviour with adwaita as well?

**Comment by cbcoutinho** [comment]
Yes Adwaita produces the same behavior - they have a default icon that is also a grey box, but fits into their theme. I guess you know about this issue then, no worries.

Thanks for the great repo

**Comment by rudrab** [comment]
Actually, it is something to do with wayland. If at the login time, you choose gnome at xorg, you will probably see everything is fine. This is a known issue in gnome, and they are probably working with that, though, for libreoffice. I will add okular to that, as well.
